lib/main/widgets/pet_list_item.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class PetListItem extends StatelessWidget {
+  final String namePet;
+  final String breedPet;
+
+  //final Image imagePet;
+
+  const PetListItem({Key? key, required this.namePet, required this.breedPet})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 27.0),
+        child: Row(
+          children: [
+            Container(
+              decoration: BoxDecoration(border: Border.all(color:const Color.fromRGBO(196, 196, 196, 1)), color: Colors.grey,),
```

Comment:
I think that there are more than 80 symbols here. Don't forget to format you code.